test/smoke_test/bcda_client.go

```diff
@@ -81,7 +81,7 @@ func getAccessToken() string {
 func startJob(resourceType string) *http.Response {
 	client := &http.Client{}
 
-	var url string = fmt.Sprintf("%s://%s/api/v1/%s/$export", proto, apiHost, resourceType)
+	var url string = fmt.Sprintf("%s://%s/api/v1/Patient/$export?_type=%s", proto, apiHost, resourceType)
```

Comment:
What about a test for `/Patient` all?

Reply:
Yeah, we'll need a test for `/Patient` all so we can validate that the response format is correct.

Reply:
it was my understanding that we would add the new test cases (including patient all) in this future ticket here: BCDA-2284 and this pr was to retrofit all existing cases to work with the new patient query parameters. I could be wrong though, let me know if this is totally not the case.

Reply:
Yes, all permutations can be addressed in BCDA-2284.  However, the standard `/Patient` request should probably be tested as part of this ticket.

Reply:
done!